### ISSUE TITLE
Decimal places - , vs . separator fallback

### ISSUE THREAD

**Issue by Eldenroot** [Sama34/newpoints#5]
Fallback for "," separator used in Europe... "," is ignored

**Comment by Sama34** [comment]
I'm unsure if I understand this correctly.

The points formatting uses MyBB code system, so it should work:
![image](https://github.com/user-attachments/assets/a81dc3b4-516b-41ef-9c28-88b0e1e8113f)
![image](https://github.com/user-attachments/assets/26f5880f-81b8-4aaa-8aa9-6651e6324d5a)

**Comment by Eldenroot** [comment]
Write 1,00 instead of 1.00 in plugin acp settings income page

**Comment by Sama34** [comment]
Oh, this seems like a limitation of the input tag for number type inputs. Is the issue merely cosmetic or visual, or does the plugin fail or breaks because of this ?

**Comment by Eldenroot** [comment]
I think that only rate is not visible in newpoints.php page. Maybe not working at all the income for the rate setting.

I just advise to add a note about using of decimal place separator in acp to make it less confusing.

**Comment by Sama34** [comment]
Did you close this on purpose ? Are the issues you mention fixed now ?

**Comment by Eldenroot** [comment]
No, but I use "." now... just like I said, there should be a note in a setting description
